Stylistic changes; expansion on a small number of commands

kfrn opened the pull request:
Style changes for readability and/or uniformity:
- add spaces after commas in long filtergraphs
- in commands, only use italics for inputs/outputs, not 'regular' variables
- inline links rather than represented like so: [[more]](#)
- various little things: grammatical fixes, YUV → Y′C<sub>B</sub>C<sub>R</sub>, etc

Command expansion:
- break out NTSC → H.26 command into subsegments
- add lines describing the filtergraph's opening and closing quotation marks for several commands

Apologies in advance for so many changes in one PR, it's quite a lot to review :(

kfrn committed:
Fix-ups: inline links, grammar corrections, etc

I've also moved the link to Dinah Handel's NDSR blog (http://ndsr.nycdigital.org/diving-in-head-first/) next to a different subcommand that it more directly relates to.

kfrn committed:
Break out commands into subsegments

I've elaborated further on certain commands, and added lines for opening and closing quotation marks for others.
There are also some minor style fixes.

kfrn committed:
Style: add spaces in long filtergraphs; only use italics for inputs

In filtergraphs that are reasonably long or unwieldy, I've added a space after each comma for readability.
In ffmpeg commands, italics should denote user inputs/outputs. Where italics had been used for 'regular' variables (i.e., not an input/output file or a variable for which the user must explicitly choose the value), I've deleted them.

These are stylistic changes that, I think, make things more readable and/or more uniform.

retokromer commented on index.html:
```diff
@@ -964,7 +975,7 @@
             <dt>-c copy</dt><dd>use stream copy mode (no re-encoding)<br>
             <dt>-map 0</dt><dd>Tells ffmpeg to map all streams of the input to the output.</dd>
-            <i>Note:</i> watch out when using <code>-ss</code> with <code>-c copy</code> if the source is encoded with an interframe codec (e.g., H.264). Since ffmpeg must split on i-frames, it will seek to the nearest i-frame to begin the stream copy.</dd>
+            <b>Note</b>: watch out when using <code>-ss</code> with <code>-c copy</code> if the source is encoded with an interframe codec (e.g., H.264). Since ffmpeg must split on i-frames, it will seek to the nearest i-frame to begin the stream copy.</dd>
```
I would let also the : as bold.
Also below.

kfrn replied:
I have an aversion to bolding colons in such positions, I don't know why.

retokromer replied:
I don’t know the rules in English. In the languages I know the interpunction signs immediately following a word are set in the same type.

kfrn replied:
It feels wrong to me, but the Chicago Manual of Style indeed indicates that the colons should be bolded. I'll change them.

retokromer commented on index.html:
```diff
@@ -1455,13 +1466,13 @@
           <dl>
             <dt>ffprobe</dt><dd>starts the command</dd>
-            <dt>-f lavfi</dt><dd>tells ffprobe to use the Libavfilter input virtual device <a href="http://ffmpeg.org/ffmpeg-devices.html#lavfi" target="_blank">[more]</a></dd>
+            <dt>-f lavfi</dt><dd>tells ffmpeg to use the <a href="http://ffmpeg.org/ffmpeg-devices.html#lavfi" target="_blank">Libavfilter</a> input virtual device</dd>
```
?
Is this an ffmpeg or ffprobe command?

kfrn replied:
Sorry, I'll correct this! There are a couple of other ones similar, as well. Ignore them, I have just fixed them. I will push them up with any other corrections you find.

retokromer commented on index.html:
```diff
@@ -2038,24 +2052,24 @@
+            <dt>"</dt><dd>quotation mark to end drawtext filter command</dd>
             <dt><i>output_file</i></dt><dd>path, name and extension of the output file.</dd>
           </dl>
+          <p>Note: <code>-vf</code> is a shortcut for <code>-filter:v</code>.</dd></p>
```
Please delete the `</dd>`.

kfrn replied:
👍

kfrn committed:
Cleanups: bold post-heading colons, ffmpeg → ffplay/ffprobe as approp.

In several places, ffprobe or ffplay commands said ffmpeg in the command breakdown. This is because of me doing a sloppy cut & paste in a previous commit in this PR.

Previously, colons succeeding a bold 'Notes' mini-heading were not themselves also bold. Reto noted this, and the Chicago Manual of Style does indicate that such colons should also be in bold.

retokromer reviewed (approved):
LGTM
Thank you very much @kfrn!

ablwr commented:
Wow thank you @kfrn and @retokromer for quick review!

FWIW I also don't bold the colon in such statements. But I don't think there's a rule.

kfrn commented:
thanks for the review @retokromer !
